lib/index.js

```diff
@@ -100,7 +100,7 @@ internals.Client.prototype.request = function (method, url, options, callback, _
         req.removeListener('error', onError);
         req.on('error', Hoek.ignore);
         clearTimeout(timeoutId);
-        self.emit('response', err, req, res, start);
+        self.emit('response', err, req, res, start, uri);
```

Comment:
It might be worth Wreck storing some of the data that isn't fully available later on, such as the URL.